mteb/models/gritlm.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from functools import partial
+
+from mteb.model_meta import ModelMeta
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+try:
+    from gritlm import GritLM
```

Comment:
I would probably pack this in the loader:

```
def gritlm_loader(**kwargs):
    from gritlm import GritLM
    return GritLM(**kwargs)
```

That way you can e.g. check what are all the models trained on.

Reply:
Agreed & updated!